website/docs/admin_hosts_houdini.md

```diff
@@ -3,20 +3,78 @@ id: admin_hosts_houdini
+
+![general_settings_self_publish_button](assets/houdini/admin/general_settings_self_publish_button.png)
+
+### Update Houdini Vars on context change
+
+Allows admins to have a list of vars (e.g. JOB) with (dynamic) values that will be updated on context changes, e.g. when switching to another asset or task.
+
+:::info
+**JOB** Var: If `JOB` is present in the Houdini var list and has an empty value, Ayon will set its value to `$HIP`
+:::
+
+![update-houdini-vars-context-change](assets/houdini/admin/update-houdini-vars-context-change.png)
+
+1. **Update Houdini vars on context change :** Disabling this setting will leave all Houdini vars unmanaged and thus no context update changes will occur.
+2. **(+) :** add new Houdini variable
+3. **Var :** The Houdini variable you want to add. No need to include `$`.
+  > For consistency reasons we always force all vars to be uppercase. e.g. `myvar` will be `MYVAR`
+4. **Value :** The value you want Ayon to maintain. 
+  > Using template keys is supported but formatting keys capitalization variants is not, e.g. `{Asset}` and `{ASSET}` won't work. For a list of available template keys go to [available template keys](#what-template-keys-are-available-for-use-in-houdini-settings)
+1. **Treat as directory :** If activated, Ayon will consider the given value as a path of a folder.
+  > If the folder does not exist on the context change it will be created by this feature so that the path will always try to point to an existing folder.
+
+## Color Management (ImageIO)
+
+Allows admins to override the global color management settings, check [Host specific overrides](https://ayon.ynput.io/docs/admin_colorspace#host-specific-overrides)
+
+![houdini-color-management](assets/houdini/admin/houdini-color-management.png)
 
 ## Shelves Manager
-You can add your custom [Shelves](https://www.sidefx.com/docs/houdini/shelf/index.html) into Houdini by setting your shelf sets, shelves and tools in **Houdini -> Shelves Manager**.
-![Custom menu definition](assets/houdini-admin_shelvesmanager.png)
+
+Allows admins to add studio's [Shelves](https://www.sidefx.com/docs/houdini/shelf/index.html) and scripts into Houdini. 
+
+### Different Modes
+
+Shelves manager works in two modes: 
+1. Add a `.shelf` file. you are required only to fill in the `Shelf Set Path`.
+  ![houdini_shelves_manager_mode_1](assets/houdini/admin/houdini_shelves_manager_mode_1.png)
+
+2. Add Scripts and Ayon will create a shelf for you.
+  > *Currently, Ayon just adds the new shelf definition in `default.shelf`*
+  
+  ![houdini_shelves_manager_mode_2](assets/houdini/admin/houdini_shelves_manager_mode_2.png)
 
 The Shelf Set Path is used to load a .shelf file to generate your shelf set. If the path is specified, you don't have to set the shelves and tools.
 
+### Shelves Example
```

Comment:
The shelves can get a bit better description that it's **either** the shelf set path or the scripts below for a particular shelf set.

So we could word it e.g. like this:

> You have two options to define a shelf which are mutually exclusive from one another:
> ﻿
> - If you set a Shelf Set Path that shelf is loaded, any scripts defined below it are ignored.
> - If you have **no** Shelf Set Path, then it'll create a shelf dynamically from the Shelf Scripts.